### PR TITLE
MEN-4382: Add deviceconfig service to the integration components

### PR DESCRIPTION
**Pull request by alfrunes**

### CODE REVIEW

**Review by lluiscampos — COMMENTED** [review]
Please do not merge this until the traefik PRs + mender-docs + the release is done :)

File: component-maps.yml [hunk 1]
```diff
@@ -486,3 +493,10 @@ docker_container:
     docker_image:
     - deviceconnect
     release_component: true
+
+  mender-deviceconfig:
+    git:
+    - deviceconfig
+    docker_image:
+    - deviceconfig
+    release_component: true
```
Comment (kacf) [inline component-maps.yml:502]: You are missing an entry under the `docker_image` top level key. These blocks always come in threes.

**Review by kacf — CHANGES_REQUESTED** [review]

File: docker-compose.config.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+version: '2.1'
+
+services:
+    mender-deviceconfig:
+        image: mendersoftware/deviceconfig:master
```
Comment (lluiscampos) [inline docker-compose.config.yml:5]: The tag here should be `mender-master`

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

File: docker-compose.config.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        image: mendersoftware/deviceconfig:mender-master
+        extends:
+            file: common.yml
+            service: mender-base
+        networks:
+            - mender
+        depends_on:
+            - mender-mongo
+        command: server --automigrate
+        labels:
```
Comment (mchalski) [inline docker-compose.config.yml:14]: I'm here mostly for this section, +1 :)

**Review by lluiscampos — APPROVED** [review]

**Comment by lluiscampos** [comment]
> Please do not merge this until the traefik PRs + mender-docs + the release is done :)

I won't block this any-more. Just coordinate with @kjaskiewiczz and the `mender-docs` test fixes.

**Comment by alfrunes** [comment]
The [QA pipeline](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/246993880) with mendersoftware/mender-qa#477 passed.
@kacf I'm awaiting your second round of review before I'm merging all the deviceconfig integration PRs.

**Review by kacf — APPROVED** [review]
Perfect! Sorry for the delay!

**Pull request merged**